src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -577,7 +577,8 @@ private static InvitationViewModel MapFrom(TeamMember teamMember)
                 Name = teamMember.Name,
                 Role = teamMember.Role,
                 Status = teamMember.Status,
-                ExpiryDate = teamMember.ExpiryDate
+                ExpiryDate = teamMember.ExpiryDate,
+                HashedAccountId = teamMember.HashedAccountId
```

Comment:
All the work was already done. Hashed accountid did not appear because it was not getting populated. I just populated this one value in the response.response.Data = MapFrom(queryResponse.TeamMember);

Reply:
Because  HashedAccountId was null..it was not getting the entire account information. Hence once i populated the HashedAccountId ..it went inside the if condition to fetch all the account information. 
EmployerTeamController.cs => line no. 331
if (model != null && model.HashedAccountId != null)

Reply:
ok thanks